uaclient/cli.py

```diff
@@ -206,20 +206,38 @@ def new_f(cmd_args, *args, **kwargs):
     return new_f
 
 
-def assert_attached(unattached_msg_tmpl=None):
-    """Decorator asserting attached config.
-
-    :param unattached_msg_tmpl: Optional msg template to format if raising an
-        UnattachedError
-    """
+def assert_attached():
```

Comment:
Just a question, why did you move the msg from here?

Reply:
Originally, the message was designed in a way that there is a generic "Unattached Error", but some commands could define their own messages.

I changed it because now there is logic to determine when to use different messages, even for a single command like `enable`. As we need different messages and specific logic, even passing a list of messages would not do, and getting the command from the args is easier than passing information through the decorator imho.